folding-schemes-solidity/templates/nova_cyclefold_decider.askama.sol

```diff
@@ -0,0 +1,24 @@
+{{ groth16_verifier }}
+
+{{ kzg10_verifier }}
+
+/**
+ * @author  PSE & 0xPARC
+ * @title   NovaDecider contract, for verifying zk-snarks Nova IVC proofs.
+ * @dev     This is an askama template. It will feature a snarkjs groth16 and a kzg10 verifier, from which this contract inherits.
+ *          WARNING: This contract is known to not be sound. It lacks checks to ensure that no soundness issues can happen.
```

Comment:
is it "known to not be sound" as it says, or is it "not known to be sound"? like, we don't know if it is sound or not (instead of knowing that it is not for sure)

Reply:
oh ok, so this refers to the contract code being incomplete, right? I was initially wrongly understanding that this was a general comment on not being properly audited, but I see now that is more regarding the code being 'unfinished' in the sense that we have pending to update the interface and the internal logic that binds the challenges, etc.